docs/topics/async.txt

```diff
@@ -116,6 +116,30 @@ mode if you have asynchronous code in your project.
 
 .. _async-safety:
 
+Async view decorators
+=====================
+
```

Comment:
maybe version added annotation needed and release notes

Reply:
What would this come under? `What's new in Django 3.X`? `Minor features`?

Reply:
I would just use your best judgment. Be sure to edit `4.0.txt`.